### ISSUE TITLE
Merge Airsonic

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#383]
Merge HEAD of Airsonic.

Mostly updated libraries.
- Includes new functions (health check)
- Check if Lucene has been upgraded and affected

Change the current 107.0.1 milestone to 107.1.0 and continue.

**Comment by file-struct** [comment]
There is a problem accessing external resources.
Temporarily skip the test.

```
[INFO] Running org.airsonic.player.service.InternetRadioServiceTest
2020-02-19 13:00:21.006  WARN   --- [           main] o.a.player.service.InternetRadioService  : Remote playlist has too many sources (maximum 250)
2020-02-19 13:00:21.027 ERROR   --- [           main] o.a.player.service.InternetRadioService  : Failed to retrieve sources for internet radio http://example.com/stream_infinity_move.m3u.
org.airsonic.player.service.InternetRadioService$PlaylistHasTooManyRedirects: Too many redirects (21) for URL http://example.com/stream_infinity_move.m3u
	at org.airsonic.player.service.InternetRadioService.connectToURLWithRedirects(InternetRadioService.java:275)
	at org.airsonic.player.service.InternetRadioServiceTest.testRedirectLoop(InternetRadioServiceTest.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.mockito.internal.runners.JUnit45AndHigherRunnerImpl.run(JUnit45AndHigherRunnerImpl.java:37)
	at org.mockito.runners.MockitoJUnitRunner.run(MockitoJUnitRunner.java:62)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
```

**Comment by tesshucom** [comment]
Inaccessible due to JWT update #386

Issued another issue due to limited scope.
Once the merge is complete, address it as another issue.

**Comment by tesshucom** [comment]
Translate Internal help into Japanese (#388)

**Comment by tesshucom** [comment]
No problem.

**Comment by tesshucom** [comment]
Adjust pop-up style (#392)
